share/revng/test/configuration/revng/decompilation.yml

```diff
@@ -11,6 +11,14 @@ commands:
     suffix: /
     command: |-
       revng artifact --resume "$OUTPUT" --analyze decompile-to-single-file "$INPUT" -o /dev/null
+  - # Run new decompilation pipeline
+    type: revng.new-decompilation-pipeline
```

Comment:
I think we should settle with either the `legacy-` prefix for the old stuff or `new-` for the new stuff.
If we don't do this now, we'll have some and some and it will be a mess.

Reply:
Fair enough. This isn't even the full new decompilation pipeline.

I'm switching the old tests to `legacy.

Reply:
Done. You can check out the last three commits of the patchset.

This also made me realize that there were a few additional end-to-end tests that could be added. I'm working on those now.

Reply:
The tests now pass. There's not much to say left here. Closing the conversation.